28_FFTBloom/main.cpp

```diff
@@ -695,7 +700,7 @@ class FFTBloomApp final : public examples::SimpleWindowedApplication, public app
 			// Compute required WorkgroupSize and ElementsPerThread for FFT
 			// Remember we assume kernel is square!
 
-			auto [elementsPerInvocationLog2, workgroupSizeLog2] = workgroup::fft::optimalFFTParameters(m_device->getPhysicalDevice()->getLimits().maxOptimallyResidentWorkgroupInvocations, kerDim.width);
+			auto [elementsPerInvocationLog2, workgroupSizeLog2] = workgroup::fft::optimalFFTParameters(m_device->getPhysicalDevice()->getLimits().maxOptimallyResidentWorkgroupInvocations, kerDim.width, m_device->getPhysicalDevice()->getLimits().minSubgroupSize);
```

Comment:
you can't just pass `minSubgroupSize`, you need to pass the actual subgroup size you will be creating the subgroup with (which IMHO should be the max size)


Furthermore why haven't you declared `m_device->getPhysicalDevice()->getLimits()` as a const reference variable somewhere to improve code readability?